ops/Vagrantfile

```
@@ -0,0 +1,37 @@
+port = ENV["HOST_PORT"] || 8000
```

When using the command, I find it more meaningful to do `PORT=8080 vagrant up` than `HOST_PORT=8080 vagrant up`.
```suggestion
port = ENV["PORT"] || 8000
```